### ISSUE TITLE
Changes in existing shedule are not carried over. 

### ISSUE THREAD

**Issue by Olli2807** [nielsfaber/scheduler-card#601]
### Checklist

- [X] I checked for similar existing issues (both open and closed) before posting.
- [X] I will participate in further discussion about this issue and can help by testing (if requested).

### Card Version

v2.3.6

### Component Version

v3.2.10

### Bug description

Note: Issue present since Update to HA Core 2022.9.1
Changes in the sheduler time are not carried over when hitting the save button.
It seems like the frontend is updated regularly to the currently saved setting.
It can easily be seen when editing an existing shedule e.g. from "Daily" to "Workdays". After some time (~5sec) it is jumping back to "Daily" and leaving the "Workdays" in light grey.
![grafik](https://user-images.githubusercontent.com/101456952/189449864-f1afc278-5955-499d-8a59-8d2736737c04.png)

Issue would not be that critical, as 5 sec is plenty of time to hit the save button and store the required changes, but after installation of an other Add-On (https://github.com/mk-maddin/wattpilot-HA) the update time of the frontend goes down to <1sec. which leafes no time for hiting the save button.

### Steps to reproduce

Edit existing schedule.
Change time with the arrows, or make any change in the Days section.
After some seeconds the change is lost. (Changed time is still displayed, but not saved, when hitting the save button. The days selection is jumping back to the previous setting.)

 

### Additional info

This is not present with HA Core 2022.8.7.
Issue first detected with HA Core 2022.9.1
Present with Chrome, Firefox and Android App.
Issue is not present when editing the number field for the time directly.

**Comment by raulliive** [comment]
The issue is reproducible also with 2022.9.0 and without your mentioned Wattpilot add-on.

**Comment by gvssr** [comment]
Reproducible for me as well.

**Comment by SpicyLimes** [comment]
I am having the same issue as well.

I am not able to add Climate Schemes any longer.

**Comment by nielsfaber** [comment]
I can confirm this bug. It is probably introduced by changes in the recent HA version.
I will have to investigate what changes I have to do to restore the previous functionality. 
To be continued..

**Comment by SpicyLimes** [comment]
Thanks for looking into it. If there is anything that we can do to contribute, please let us know.

As a side note, I used my iPhone to attempt to re-create an HVAC schedule (since it relies on touch rather than a mouse) and was able to successfully re-create a schedule. It was not easy, and there were very short delays where a selection was made and the save button was pressed, but it worked. 

I then used my old Microsoft Surface laptop (which features touchscreen support) to re-create another light schedule and I can confirm it is much easier to complete a schedule with the combination of a mouse and touchscreen. 

Basically what I am saying is that you have to be very quick when adjusting options/settings, then saving it, within the scheduler card - but it is doable.

**Comment by nielsfaber** [comment]
This issue should be fixed by updating the card to v2.3.7.
@Olli2807  Please try it out and close the issue if so. Otherwise let me know.

**Comment by Olli2807** [comment]
Can confirm, that Issue is solved with v3.2.7.
Thank you so very much. 
Still playing around with the scheduler, but  its on its way to my most prefered addon.

**Comment by Zebble** [comment]
I'm having the exact same issue , even with 2.3.7.

HA 2022.8.6

Anything I can provide to help troubleshoot?

**Comment by Zebble** [comment]
Strange...

2.3.7 was showing as installed but I tried a "redownload" to be sure.  Now it's working!
